docs/utils/SolidityLexer.py

```diff
@@ -54,11 +54,10 @@ class SolidityLexer(RegexLexer):
              r'(<<|>>>?|==?|!=?|[-<>+*%&\|\^/])=?', Operator, 'slashstartsregex'),
             (r'[{(\[;,]', Punctuation, 'slashstartsregex'),
             (r'[})\].]', Punctuation),
-            (r'(for|in|while|do|break|return|continue|switch|case|default|if|else|'
-             r'throw|try|catch|finally|new|delete|typeof|instanceof|void|'
-             r'this|import|mapping|returns|private|public|external|internal|'
-             r'constant|memory|storage|payable)\b', Keyword, 'slashstartsregex'),
-            (r'(var|let|with|function|event|modifier|struct|enum|contract|library)\b', Keyword.Declaration, 'slashstartsregex'),
+            (r'(for|in|while|break|return|continue|if|else|throw|new|delete|this|super|'
```

Comment:
I'm not sure about `this` and `super`, as they do seem like a keyword, but technically they are not declared as a keyword.